app/api/helpers/storage.py

```diff
@@ -162,7 +162,9 @@ def upload_local(uploaded_file, key, **kwargs):
         return get_settings()['static_domain'] + \
             file_relative_path.replace('/static', '')
     url = urlparse(request.url)
-    return url.scheme + '://' + url.hostname + file_relative_path
+    return '{scheme}://{hostname}:{port}{file_relative_path}'.format(
```

Comment:
If the port is 80 and scheme is http and if the port is 443 and scheme is https, we can omit the :port 

I think this should be accomodated